### ISSUE TITLE
Command Line Interface on Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by fahmyadan** [huawei-noah/SMARTS#1677]
**High Level Description**
I have been using SMARTS for research smoothly on my previous OS (Ubuntu 20.04). Since upgrading to to 22.04 LTS, I am unable to pass the sanity tests, specifically by not being able to access the command line software collections tool scl.  

**Desired SMARTS version**
Main branch 

**Operating System**
Ubuntu 22.04 LTS

**Problems**
When I run `make sanity-tests` after install packages using pip in my virtual env, The terminal returns:
`scl scenario build-all scenarios`
  `make : scl: No such file or directory` 
`make: *** [Makefile:74: build-all-scenarios] Error 127`

I have tried installing scl manually using `yum install scl-utils scl-utils-build` but I am aware that yum is no longer supported for my OS version so that doesn't work either.

I would be grateful if someone could advise on whether there is a fix for this or whether I should go back to using SMARTS in Ubuntu 20.04.

Thanks in advance!

**Comment by Gamenot** [comment]
Hello @fahmyadan I am not quite certain what the issue might be.

There have not been many tests of `22.04` yet but the `scl` command is a the command `python -m cli.cli` which should be in a `cli` module installed with the `smarts` package.

It is entirely possible that there is a module name conflict we should change the `cli` module name.

**Comment by fahmyadan** [comment]
Thanks @Gamenot for clearing that up. I made some slight changes to the Makefile to accommodate the alias which allows me to now build scenarios. 

After installing the necessary packages manually, I have made some progress on `make sanity-test` , though I am now getting the following:
`ERROR:: usage: pytest [options] [file_or_dir] [file_or_dir] [...]`
`pytest: error: unrecognized arguments: --forked --dist=loadscope -n`
`inifile: None`
`make: *** [Makefile:24: sanity-test] Error 4`

It's unclear to me where the problem is happening. I look forward to your response. 

Thanks in advance. 




`
